lib/velum/salt_minion.rb

```diff
@@ -9,7 +9,7 @@ class SaltMinion
     attr_accessor :minion_id
 
     ROLES_MAP = {
-      master: ["kube-master", "etcd"],
+      master: ["kube-master"],
```

Comment:
Right now all minions will be etcd-proxies but in the future we might need to revisit this one. Looks good for now.